src/components/SidebarTabs/SharingSidebarTab.vue

```diff
@@ -232,6 +252,11 @@ export default {
 		return {
 			isLoading: false,
 			appConfig: loadState(appName, 'appConfig'),
+			qrModal: {
+				on: false,
+				text: '',
+				uri: '',
+			}
```

Comment:
```suggestion
			},
```
Missing trailling comma to silence the lint warning